doc/api/fs.md

```diff
@@ -3072,6 +3072,32 @@ extended part is filled with null bytes (`'\0'`):
 
 If `len` is negative then `0` will be used.
 
+The `fs.read()` method reads data from the file specified by the file
+descriptor (`fd`). The `length` argument indicates
+the maximum number of bytes that Node.js will attempt
+to read from the kernel. However, the actual
+number of bytes read (`bytesRead`) can be lower
+than the specified `length` for various reasons.
+
+For example:
+
+* If the file is shorter than the specified `length`, `bytesRead`
+  will be set to the actual number of bytes read.
+* If the file encounters EOF (End of File) before the buffer could be filled,
+  Node.js will read all available bytes until EOF is encountered, and the
+  `bytesRead` parameter in the callback will indicate the actual number of
+  bytes read, which may be less than the specified `length`.
```

Comment:
The updated version doesn't elaborate difference between "shorter files" and "reaching EOF" (e.g. by providing example with pipe/socket/device node/etc.).
For regular files under normal circumstances it's essentially the same, so IMHO it's safe to omit one of these.

Reply:
I would be very happy if you suggest an arrangement for this place, I don't know how to do it right now.

Reply:
Either just omit one of these, or reword the whole thing.
Something among the lines "The actual number of bytes read can be lower than specified `length` for various reasons, such as reaching end of file or reading from slow network filesystem" / "It might be needed to handle such cases, for example, with loop" / "This is equivalent to POSIX `preadv2()`" should suffice IMHO.

Reply:
Either just omit one of these, or reword the whole thing.
Something among the lines "The actual number of bytes read can be lower than specified length for various reasons, such as reaching end of file or reading from slow network filesystem" / "It might be needed to handle such cases, for example, with loop" / "This is equivalent to POSIX preadv2()" should suffice IMHO.

Reply:
I followed what you said ❤️